pandas/tests/tools/test_to_numeric.py

```diff
@@ -649,3 +649,14 @@ def test_failure_to_convert_uint64_string_to_NaN():
     ser = Series([32, 64, np.nan])
     result = to_numeric(pd.Series(["32", "64", "uint64"]), errors="coerce")
     tm.assert_series_equal(result, ser)
+
+
+def test_support_downcast_of_nullable_dtypes():
+    # GH 33013
+    try:
+        pd.to_numeric(pd.Series([1, 2, 3], dtype="Int32"), downcast="integer")
+        pd.to_numeric(pd.Series([1, 2, 3], dtype="Int64"), downcast="integer")
+        pd.to_numeric(pd.Series([1, 2], dtype="Int32"), downcast="signed")
+        pd.to_numeric(pd.Series([1, 2, 3], dtype="Int32"), downcast="float")
+    except TypeError:
```

Comment:
Rather than the try...except you would ideally do `expected = ...` and do `tm.assert_frame_equal(result, expected)` for the various cases; you will see this throughout other tests if you need a place to look